.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Flink.groovy

```diff
@@ -31,7 +31,7 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java_ValidatesRunner_Flink',
   // Publish all test results to Jenkins
   publishers {
     archiveJunit('**/build/test-results/**/*.xml')
-  }
+  }metconstep
```

Comment:
Is this needed?

Reply:
oh, that's a typo! thanks, let me fix now

Reply:
pushed a corrected version, good catch!

that's me trying to search for the `MetricsContainerStepMap` file when IntelliJ is not ready, and has a race condition with the file-search popup appearing and the text going there 😞